store/tikv/coprocessor.go

```diff
@@ -770,7 +771,7 @@ func (worker *copIteratorWorker) calculateRemain(ranges *copRanges, split *copro
 
 func (it *copIterator) Close() error {
```

Comment:
Will Close method be called multiple times?

Reply:
No, but the `it.finished` channel will be closed somewhere else.